AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.signalrservice/signalr/config.json

```
@@ -9,42 +9,40 @@
     "searchSuffix": "signalr",
     "azureServiceName": "Azure SignalR Services",
     "armApiConfig": {
-		"armApiVersion": "2020-05-01"
-	},
+        "armApiVersion": "2020-05-01"
+    },
     "isSearchEnabled": true,
-	"keystoneDetectorId": "",
+    "keystoneDetectorId": "",
     "liveChatConfig": {
         "isApplicableForLiveChat": false,
         "supportTopicIds": []
     },
     "categories": [
         {
-            "id": "OverallCheck",
-            "name": "Overall Check",
-            "description": "An overall check run to help quickly identify some possible common issues. ",
-            "keywords": [ "Overall Check" ],
-            "color": "rgb(186, 211, 245)",
-            "createFlowForCategory": false,
-            "chatEnabled": false
-        },
-        {
-            "id": "AvailabilityAndPerformance",
+            "id": "AvailabilityAndPerformanceSignalR",
             "name": "Availability And Performance",
             "description": "Investigate issues with your azure signalr service. Find out the common causes and investigate in detail. ",
-            "keywords": [ "Config Sever", "App Health", "Availability" ],
+            "keywords": [ "Config Sever", "App Health", "Availability", "sdk" ],

```

Since availability is the title of the category, can we remove it to reduce redundancy and replace it with "Error Rate"? Also, I changed the last keyword to be in all caps.
```suggestion
            "keywords": [ "Config Sever", "App Health", "Error Rate", "SDK" ],

```